dali/dfu/dfuserver.cpp

```diff
@@ -149,9 +149,14 @@ int main(int argc, const char *argv[])
     if (!stop) {
         sentinelFile.setown(createSentinelTarget());
         removeSentinelFile(sentinelFile);
+        int maxdetail = 1000;
+#ifndef _CONTAINERIZED
         Owned<IComponentLogFileCreator> lf = createComponentLogFileCreator(globals, "dfuserver");
-        lf->setMaxDetail(1000);
+        lf->setMaxDetail(maxdetail);
         fileMsgHandler = lf->beginLogging();
+#else
+        updateContainerizedLogMsgHandler(getCategoryLogMsgFilter(MSGAUD_all, MSGCLS_all, maxdetail));
```

Comment:
This is a component we haven't containerized.. 
In fact it's not so much that we haven't containerized, but that it hasn't been moved over to the new config file paradigm (i.e. it doesn't call loadConfiguration, so queryComponentConfig() would fire an exception).

And so for now for any component like this that hasn't been contaerized (and hasn't moved over to the new config), I would leave it untouched.